app/src/main/java/org/openobservatory/ooniprobe/activity/ResultDetailActivity.java

```diff
@@ -26,95 +42,146 @@
+    }
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        result = SQLite.select().from(Result.class)
+                .where(Result_Table.id.eq(getIntent().getIntExtra(ID, 0))).querySingle();
+        assert result != null;
+        setTheme(result.getTestSuite().getThemeLight());
+        setContentView(R.layout.activity_result_detail);
+        ButterKnife.bind(this);
+        setSupportActionBar(toolbar);
+        ActionBar bar = getSupportActionBar();
+        if (bar != null) {
+            bar.setDisplayHomeAsUpEnabled(true);
+            bar.setTitle(result.getTestSuite().getTitle());
+        }
+        pager.setAdapter(new ResultHeaderAdapter());
+        tabLayout.setupWithViewPager(pager);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(this);
+        recycler.setLayoutManager(layoutManager);
+        recycler.addItemDecoration(new DividerItemDecoration(this, layoutManager.getOrientation()));
+        result.is_viewed = true;
+        result.save();
+        items = new ArrayList<>();
+        adapter = new HeterogeneousRecyclerAdapter<>(this, items);
+        recycler.setAdapter(adapter);
+        snackbar = Snackbar.make(coordinatorLayout, R.string.Snackbar_ResultsSomeNotUploaded_Text, Snackbar.LENGTH_INDEFINITE)
+                .setAction(R.string.Snackbar_ResultsSomeNotUploaded_UploadAll, v1 -> runMKCollectorResubmitSettingsAsyncTask());
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        load();
+    }
+
+    private void runMKCollectorResubmitSettingsAsyncTask() {
+        new MKCollectorResubmitSettingsAsyncTask(this).execute(result.id, null);
+    }
+
+    private void load() {
+        boolean isPerf = result.test_group_name.equals(PerformanceSuite.NAME);
+        items.clear();
+        for (Measurement measurement : result.getMeasurements())
+            items.add(isPerf && !measurement.is_failed ?
+                    new MeasurementPerfItem(measurement, this) :
+                    new MeasurementItem(measurement, this));
+        adapter.notifyTypesChanged();
+        if (SQLite.selectCountOf().from(Measurement.class).where(
+                Measurement_Table.is_uploaded.eq(false),
+                OperatorGroup.clause()
+                        .or(Measurement_Table.is_failed.eq(false))
+                        .or(Measurement_Table.report_id.isNull()),
+                Measurement_Table.result_id.eq(result.id)).longValue() != 0)
+            snackbar.show();
+        else
+            snackbar.dismiss();
+    }
+
+    @Override
+    public void onClick(View v) {
+        Measurement measurement = (Measurement) v.getTag();
+        ActivityCompat.startActivity(this, MeasurementDetailActivity.newIntent(this, measurement.id), null /*ActivityOptionsCompat.makeClipRevealAnimation(v, 0, 0, v.getWidth(), v.getHeight()).toBundle()*/);
```

Comment:
why there is a commented line here?

Reply:
is only a placeholder for removed transition